api/src/guards/admin-no-pii.guard.ts

```diff
@@ -0,0 +1,15 @@
+import { CanActivate, ExecutionContext, Injectable } from '@nestjs/common';
+import { User } from '../dtos/users/user.dto';
+
+@Injectable()
+export class AdminNoPIIGuard implements CanActivate {
```

Comment:
I'm finding the naming of this to be confusing - it's a guard for no PII, but does allow limited jurisdictional admins?